.travis.yml

```diff
@@ -8,6 +8,8 @@ julia:
   - 1.1
   - 1.2
   - 1.3
+  - 1.4
+  - 1.5
```

Comment:
Thanks! These are exactly the right lines to change, but I don't think we need to test every version, and to save us having to add new versions in the future we can just use `1` which will get us the latest stable  `1.x` version (so 1.4 right now, but 1.5 soon)
```
julia:
  - 1.0   # long-term support version
  - 1     # latest stable version
  - nightly
```